### PR TITLE
Propose to clarify

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint/issues/3303

### CODE REVIEW

File: migration/mover-retirement-timeline.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ description: "Mover retirement timeline"
 # Mover.io retirement timeline
 
 >[!Important]
->**Attention businesses or those with EDU plans**: We will soon be retiring the legacy [Mover](https://app.mover.io) tool's ability to migrate from [Google Drive](mm-google-overview.md), [Dropbox](mm-dropbox-overview.md), [Box](mm-box-overview.md), and [Egnyte](mm-egnyte-overview.md). Please use [Migration Manager](https://aka.ms/ODSP-MM) going forward.
+>**Attention businesses or those with EDU plans**: We will soon be retiring the legacy [Mover](https://app.mover.io) tool's ability to migrate from [Google Drive](mm-google-overview.md), [Dropbox](mm-dropbox-overview.md), [Box](mm-box-overview.md), and [Egnyte](mm-egnyte-overview.md). Please use [Migration Manager](https://aka.ms/ODSP-MM), located in the SharePoint Admin Center, going forward.
```
Comment (JohanFreelancer9) [inline migration/mover-retirement-timeline.md:21]:
```suggestion
>**Attention businesses or those with EDU plans**: We will soon be retiring the legacy [Mover](https://app.mover.io) tool's ability to migrate from [Google Drive](mm-google-overview.md), [Dropbox](mm-dropbox-overview.md), [Box](mm-box-overview.md), and [Egnyte](mm-egnyte-overview.md). Please use [Migration Manager](https://aka.ms/ODSP-MM), located in the SharePoint admin center, going forward.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@JoanneHendrickson PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@JoanneHendrickson please help me with the review

**Pull request merged**